src/table/table.scss

```diff
@@ -16,75 +15,205 @@
     box-sizing: border-box;
   }
 
-  > .iui-tables-body {
-    width: 100%;
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
+  .iui-table-header {
+    @include iui-table-header;
+  }
+
+  .iui-table-body {
+    @include iui-table-body;
+  }
+
+  .iui-cell {
```

Comment:
So this duplicates rules for cell in body, as body mixin has cell rules there..
And it is not that we have cell under main table (it is under header and body).
So not sure, maybe handle cell in there versus having it here (but then of course, nesting..)

Reply:
When I structure the table initially, I made it with this nesting. So what you are asking is because you define nesting was wrong. 
## Structure :
- iui-table
  - iui-header (Style for header only)
    - iui-row
      - iui-cell / iui-sorted / iui-filtered / iui-action
        - iui-value
        - iui-filter
        - iui-sort/iui-descending
        - iui-resizer
  - iui-body (Style for Body only)
    - iui-row / iui-selected / iui-new / iui-disabled
      - iui-cell
        - iui-value
  - iui-footer (Style for footer, does not have much in common with body and header)
    - iui-pagination
    - iui-location
  - iui-row (Global style apply to both header and body)
    - iui-cell (Global style apply to both header and body)

## Mixin :
- iui-table-size (default, condensed, extra-condensed styles)
- iui-table-special-styles (Zebra stripping, fixed height)
- iui-table-special-cell (User-icon, Progress Spinner, iui-action, checkbox)
- iui-new-green-dot (green dot for new, not sure this need to be a mixin)

Reply:
I think it would be better to have mixin for repeated rules than to have this:
![image](https://user-images.githubusercontent.com/81580355/120648000-dd731a00-c483-11eb-9c26-8e8589986cbd.png)